test/common/testinsp/services.py

```diff
@@ -0,0 +1,20 @@
+from testinsp.base import TestInspector
+from testinsp.constants import PLAIN, STORE_PATH
+
+
+class ServiceInfo(TestInspector):
+    store_type = PLAIN
+    _get_data_command = "systemctl --type=service --state=running"
```

Comment:
This is a *lot* of data, and just overwhelming. The vast majority of services have nothing to do with a particular test case, and just add a lot of noise to the failure logs. This makes more sense for specific services which are unexpectedly running/not running, such as `udisks.service` failing in the storage tests. But we already see all of this in the journal attachments, so it's redundant.